rules/manager.go

```diff
@@ -181,7 +182,7 @@ func (g *Group) run() {
 		g.Eval(start)
 
 		iterationDuration.Observe(time.Since(start).Seconds())
-		g.setEvaluationTimeSeconds(time.Since(start).Seconds())
+		g.SetEvaluationTime(model.Duration(time.Since(start)))
```

Comment:
Why `model.Duration` over `time.Duration`? The only difference is basically the custom formatting we have for the former. I'd like to avoid it in all places where we do not need this.